CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## BOLT NEXT
+
+### New features
+
+* **Debug logging for file transfer** ([#1256](https://github.com/puppetlabs/bolt/issues/1256))
+
+  When a file is transferred to a target the source and destination are logged at debug level.
```

Comment:
@donoghuc -  was this not being logged before or was it being logged at a level other than debug? Can you clarify in the text and re-push?